glue/viewers/image/qt/data_viewer.py

```diff
@@ -63,7 +64,7 @@ class ImageViewer(MatplotlibDataViewer):
     tools = ['select:rectangle', 'select:xrange',
              'select:yrange', 'select:circle',
              'select:polygon', 'image:contrast_bias',
-             'save:python']
+             'image:point_selection', 'save:python']
```

Comment:
Can you put it just *before* contrast_bias?

Reply:
sure! How are you ordering them? they are alphabetical now right?